setup.py

```diff
@@ -8,6 +8,8 @@
         "pytest-xdist",  # multi-process runner
         "pytest-cov",  # Coverage analyzer plugin
         "hypothesis>=6.2.0,<7.0",  # Strategy-based fuzzer
+        "ape-alchemy>=0.6.5,<0.7.0",
```

Comment:
**suggestion**:  You don't need to pin these; the plugins are already pinned to core ape so it'll grab the same version as ape automagically - less we have to update in the future this way!